parser/keys.go

```diff
@@ -69,9 +69,13 @@ func keyAdd(argv []string, cmdr cmd.Commander) error {
 	usage := `
 Adds SSH keys for the logged in user.
 
-Usage: deis keys:add [<key>]
+Usage: deis keys:add [<name>] [<key>]
+
+<name> and <key> can be used in either order and are both optional
```

Comment:
Sorry, jumped the gun on that review. It looks like we'll need to manually test this new CLI functionality to ensure backwards compatibility and if this statement is true.

So if someone does `deis keys:add ~/.ssh/id_rsa`, it should still work according to this statement, oui?